lib/api/controllers/authController.js

```diff
@@ -14,144 +20,104 @@ function AuthController (kuzzle) {
   /**
    * Logs the current user out
    *
-   * @param {RequestObject} requestObject
-   * @param {Object} userContext
-   * @returns {Promise}
+   * @param {Request} request
+   * @returns {Promise<Object>}
    */
-  this.logout = function authLogout (requestObject, userContext) {
-    var
-      modifiedData = null;
-
-    return kuzzle.pluginsManager.trigger('auth:beforeLogout', {requestObject, userContext})
-      .then(data => {
-        modifiedData = data;
-
-        return kuzzle.repositories.token.expire(modifiedData.userContext.token);
-      })
-      .then(() => kuzzle.pluginsManager.trigger('auth:afterLogout', {
-        responseObject: new ResponseObject(modifiedData.requestObject, {}),
-        userContext: modifiedData.userContext
-      }));
+  this.logout = function authLogout (request) {
+    return kuzzle.repositories.token.expire(request.context.token)
+      .then(() => Promise.resolve({}))
```

Comment:
Nope: it forces resolving an empty object instead of `undefined`, which would be rejected by `Request.setResult` in the funnel controller.